src/Widgets/AppEntry.vala

```diff
@@ -71,12 +71,15 @@ public class Notifications.AppEntry : Gtk.ListBoxRow {
             active = true
         };
         unowned var expander_style_context = expander.get_style_context ();
-        expander_style_context.add_provider (provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
         expander_style_context.add_class ("image-button");
```

Comment:
You need to add another style class here like `.expander` otherwise all the other toggle buttons in the panel will also get rotated icons :sweat_smile: 

![Screenshot from 2024-10-09 15 30 15](https://github.com/user-attachments/assets/6e4bc888-c65e-4cb8-9521-16676f0f5e53)

Reply:
Sorry, I'm just getting familiar with architecture.